### ISSUE TITLE
TTS training problems: Training jets failed

### ISSUE THREAD

**Issue by JiaYK** [espnet/espnet#4432]
Hi!
I used jets to run on the aishell3 dataset, but the text information was not aligned(the synthesized speech has no semantic information).
I have tried other models (such as joint fs2 hifigan, tacotron2, Vits) that can be aligned. 
Is this normal? Jets' segmentation model will be more difficult to train.
Both use spid instead of xvector and gst.

Looking forward to your reply!

**Comment by imdanboy** [comment]
If alignment loss doesn't drop in first few epoch, how about scaling an alignment loss term by tuning `lambda_align`?
I have experienced an alignment issue in vctk datset so i tuned it 5.0 as far as i remember.

**Comment by JiaYK** [comment]
@kan-bayashi  Thank you very much for your reply.
alignment loss in first step is generator_align_loss=29.134,
in 989 epoch 601-650 batch is  generator_align_loss=24.842.
I think it's a good idea to scaling  alignment loss.
On the other hand, I trained aishell3 to use an rtx3090, so the batch_bins can only be reduced to 1200000, Should I increase training epoch to get better results (for vocoder)?

**Comment by kan-bayashi** [comment]
> On the other hand, I trained aishell3 to use an rtx3090, so the batch_bins can only be reduced to 1200000, Should I increase training epoch to get better results (for vocoder)?

In my

**Comment by JiaYK** [comment]
Thank you very much for your experience. I changed alignment loss and retrained it. I hope the model can converge well.

**Comment by seastar105** [comment]
@JiaYK could you success to train multi speaker model?

**Comment by JiaYK** [comment]
To be honest, the training was not successful. I changed the parameter of ```lambda_align``` several times, but in the end, the alignment effect was very poor. There was no semantic information. I used aishell3 for testing. Maybe the data is not very good, which is also one reason.

**Comment by seastar105** [comment]
It's sad to hear failure. Maybe small(relatively) batch_bins affect training. I had better result with bigger batch_bins in my experiment(500000 vs 2000000) on finetuning setting. I'm gonna train on multi speaker dataset. I'll report if i get nice results

**Comment by JiaYK** [comment]
The batch bins I use is 1200000 by one 3090. I look forward to your good news

**Comment by seastar105** [comment]
@yurtmete I've not started training. since I'm busy recently.. I'm gonna try KAIST AUDIOBOOK DATASET which has 11 speakers(6 male, 5 female maybe?). It's korean dataset

**Comment by hermanseu** [comment]
@imdanboy   hi, when using jets to train a single spk model,  I get good pitch loss, energy loss and duration loss, but the mel loss is non convergence, oscillating about the first output. After training 50 epochs, the mel loss is still non convergence. Are there any training strategies for jets?  Or can you please give me some advice?

**Comment by imdanboy** [comment]
@hermanseu Have you checked an alignment loss or a plotting of alignment matrix? I think abnormal alignment could be one reason for non convergence of mel loss. Whenever diagonal align matrix is plotted, I could hear intelligible sound at least.
